proxy/README.md

```diff
@@ -0,0 +1,52 @@
+# OPEA Pipeline Proxy
+
+OPEA Pipeline Proxy is an enhancement of the default Istio proxy with additional features designed specifically for OPEA RAG pipelines.
+
+## Features
+
+- [Guardrails](./deployments/guardrails/README.md)
+
+## Build
+
+OPEA Pipeline Proxy is based on Istio proxy and Envoy, you can refer to [Building Envoy with Bazel](https://github.com/envoyproxy/envoy/blob/main/bazel/README.md) for build locally. In addition, Clang and OpenVINO is required to build OPEA Pipeline Proxy.
+
+```sh
+# Build the binary locally. The binary will be generated to `bazel-bin/envoy`.
+bazel build -c opt envoy
+
+# Build the Docker image locally.
+mkdir -p bin
+cp -f bazel-bin/envoy bin/envoy
+docker build -f tools/Dockerfile .
+```
+
+(Recommended) You can also build OPEA Pipeline Proxy in the build container.
+
+```sh
+# Build the binary within the Docker. The binary will be generated to `bin/envoy`.
+tools/build.sh
+
+# Build the Docker image within the Docker. The image will be tagged as `opea/proxyv2:<ISTIO_TAG>`.
+BUILD_IMAGE=1 tools/build.sh
```

Comment:
Can you explian the ISTIO_TAG arg? And we can add it to the build command line so that users can specify the istio version: 
`BUILD_IMAGE=1 ISTIO_TAG=1.23.0 tools/build.sh`
It also applies to building locally.

Reply:
It is not a good idea to change the default tag since the whole build system (Bazel files, scripts, build image, proxy image) binds to a specific version of Istio. https://github.com/opea-project/GenAIInfra/blob/5768ac2bb3b4aad6fe48e5f994a908f4ee4f2312/proxy/WORKSPACE#L5-L8

Reply:
So currently, we doesn't want to expose a way to users to change the Istio version, right?

Reply:
Yes, it have to mount to a specific version at this point.

A tool script may be added later to auto update these fields according to the Istio version.